app/views/posts/_featured_post.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if post && post.featured? %>
+  <div>
+    <h2><%= post.title %></h2>
+    <p>** Featured **</p>
+    <p><%= post.published_at %></p>
+    <p><%= post.tags.map(&:name).join(", ") %></p>
+    <p><%= post.article.content %></p>
+    <a href="#">See full post >></a>
```

Comment:
What do you think about using a unicode right arrow character "→" instead of ">>"?

Reply:
This is just temporary. It's not going to be part of the design once that piece is done.